modules/insights-operator-enable-ofuscation.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+// * support/insights-operator/disabling-insights-operator.adoc
+
```

Comment:
support/insights-operator/disabling-insights-operator.adoc should be support/remote_health_monitoring/opting-out-of-remote-health-reporting.adoc